### ISSUE TITLE
[2.6] Multiple speakers should be transcribed simultaneously for a better accessibility

### ISSUE THREAD

**Issue by hiroshisuga** [bigbluebutton/bigbluebutton#17647]
**Describe the bug**
It's not a bug but I feel uncomfortable that only one speaker can be transcribed to show the closed caption.

**To Reproduce**
1. two or more people join a meeting with the microphone on
2. everybody turns on the speech transcription when s/he starts the audio
3. turn the captioning on and all speak simultaneously
4. See only one speaker is chosen and transcribed, as if all the others are silent.

**Expected behavior**
If many people are speaking simultaneously, they should be all transcribed. Otherwise deaf people misunderstand the conversation.

**BBB version:**
2.6.1

**Additional context**
#17571 is related anyhow. There I pointed out that the captioning does not start immediately when a user turns it on. S/he needs to speak for a while to be recognised as a speaker. I would expect everybody who turns on the captioning should be recognised imediately as a potential speaker who needs to be captioned when a speaking starts.
